test/distributed/test_c10d_functional_native.py

```diff
@@ -84,7 +88,7 @@ def _init_process_group(self) -> None:
         torch._C._distributed_c10d._register_process_group("default", dist.group.WORLD)
 
     @skip_if_lt_x_gpu(2)
```

Comment:
dont you need to kill the skip_if_lt_x_gpu(2) as well to get what you want about not requiring 2GPUs in ci?

Reply:
Oh I'm only changing the Inductor tests to use fake pg. These tests here use real nccl pg.